packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -156,7 +156,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
           <SelectionZone selection={ this.selection } selectionMode={ SelectionMode.multiple }>
             <div className={ css('ms-BasePicker-text', styles.pickerText) } role={ 'list' }>
               { this.renderItems() }
-              <BaseAutoFill
+              { this.okToAddMore() && (<BaseAutoFill
                 { ...inputProps as any }
```

Comment:
That's what I'm trying to achieve on line 246-248,  seems to me that resetFocus() handles focus being set correctly and is called from several places. you think I should put  some check in in componentwillupdate as well?